cli/drivers/WordpressComposerInstallValetDriver.php

```diff
@@ -0,0 +1,75 @@
+<?php
+
+class WordpressComposerInstallValetDriver extends ValetDriver
+{
+    /**
+     * Determine if the driver serves the request.
+     *
+     * @param  string  $sitePath
+     * @param  string  $siteName
+     * @param  string  $uri
+     * @return void
+     */
+    public function serves($sitePath, $siteName, $uri)
+    {
+        if (file_exists($composerFile = $sitePath .'/composer.json')) {
+            if (file_exists($sitePath .'/wp-config.php')) {
+                return true;
+            }
+
+            $composerJson = json_decode(file_get_contents($composerFile));
+
+            if (isset($composerJson->extra)) {
+                $extra = (array) $composerJson->extra;
+
+                if (isset($extra['wordpress-install-dir'])) {
+                    return true;
+                }
+            }
+        }
+
+        return false;
+    }
+
+    /**
+     * Determine if the incoming request is for a static file.
+     *
+     * @param  string  $sitePath
+     * @param  string  $siteName
+     * @param  string  $uri
+     * @return string|false
+     */
+    public function isStaticFile($sitePath, $siteName, $uri)
+    {
+        if (file_exists($staticFilePath = $sitePath . $uri) && !$this->fileIsPHP($uri) && !is_dir($staticFilePath)) {
```

Comment:
Tried `$this->isActualFile()` originally. For whatever reason, it throws a Fatal error: Call to undefined method for me.

Also, the `!is_dir($staticFilePath)` check is needed for any path that doesn't explicitly load the index.php such as as the dashboard landing (i.e. example.com/wp-admin/)